### PR TITLE
Update MacOS build prerequisites for manual

### PR DESCRIPTION
**Pull request by NhatDinh**

### CODE REVIEW

File: docs/manual/troubleshooting.tex [hunk 1]
```diff
@@ -856,12 +856,17 @@ \subsection{Install prerequisites\label{building-prerequisites}}
...
 brew cask install java
 brew cask install mactex
+\begin{Verbatim}
+
+Make latex directory and copy hevea.sty file into it.
+
+\begin{Verbatim}
 mkdir -p $HOME/Library/texmf/tex/latex
 cp -p /usr/local/Cellar/hevea/2.31/lib/hevea/hevea.sty $HOME/Library/texmf/tex/latex/
+Note: If copy permission denied, try sudo.
```
Comment (smillst) [inline docs/manual/troubleshooting.tex]: This text doesn't belong inside verbatim.

File: docs/manual/troubleshooting.tex [hunk 2]
```diff
@@ -856,12 +856,17 @@ \subsection{Install prerequisites\label{building-prerequisites}}
 
 \begin{Verbatim}
 brew update
-brew install caskroom/cask/brew-cask
 brew install git ant hevea maven mercurial librsvg unzip make
 brew cask install java
 brew cask install mactex
+\begin{Verbatim}
```
Comment (smillst) [inline docs/manual/troubleshooting.tex]: I think this should be `\end{Verbatim}.

File: docs/manual/troubleshooting.tex [hunk 3]
```diff
@@ -856,12 +856,17 @@ \subsection{Install prerequisites\label{building-prerequisites}}
 
 \begin{Verbatim}
 brew update
-brew install caskroom/cask/brew-cask
```
Comment (smillst) [inline docs/manual/troubleshooting.tex]: Why isn't this required anymore?

Comment (NhatDinh) [reply docs/manual/troubleshooting.tex]: brew-cask is already installed if you have Homebrew. 
$ brew install caskroom/cask/brew-cask will resolve in 
Error: Cask 'brew-cask' is unavailable: {HOME}/Homebrew/Library/Taps/caskroom/homebrew-cask/Casks/brew-cask.rb' does not exist.

**Comment by smillst** [comment]
If you merge typetools/master into your branch, the Travis tests should pass.

**Comment by NhatDinh** [comment]
Let me know if any further changes are needed.
Thank you.

File: docs/manual/troubleshooting.tex (hunk 3 above)
Comment (smillst) [reply docs/manual/troubleshooting.tex]: Thanks for the information!

**Pull request merged**